Update README.md

PricelessRabbit opened the pull request:
- fixed wrong mainflux url in doc
- added Agent start instruction in case of mainflux ui is used on the cloud

nmarcetic commented:
@PricelessRabbit Please [sign-off commit](https://github.com/mainflux/agent/pull/45/checks?check_run_id=876607739), DCO is failing. Thanks!

drasko commented:
@mteodor please review

mteodor commented on README.md:
```diff
@@ -39,7 +39,15 @@ Start Agent with:
 MF_AGENT_BOOTSTRAP_ID=<bootstrap_id> \
 MF_AGENT_BOOTSTRAP_KEY=<bootstrap_key> \
-MF_AGENT_BOOTSTRAP_URL=https://mainflux.com/bs/things/bootstrap \
+MF_AGENT_BOOTSTRAP_URL=https://mainflux.com/things/bootstrap \
```
this must be `http://mainflux.com:8204/things/bootstrap` if `bootstrap` is started additionally to the default docker composition ( and there is no mapping in nginx.conf for the bootstrap endpoint )

drasko replied:
Wouldn't it be better to add mapping for this in NginX?

manuio commented:
@PricelessRabbit DCO is failing

drasko commented:
@PricelessRabbit please update this PR according to remarks

drasko commented:
@PricelessRabbit please update the branch so we can merge.

@mteodor is this PR OK for merge? Please review.

mteodor commented:
> @PricelessRabbit please update the branch so we can merge.
> 
> @mteodor is this PR OK for merge? Please review.

remarks not addressed

> @PricelessRabbit please update the branch so we can merge.
> 
> @mteodor is this PR OK for merge? Please review.

remarks not addressed

manuio commented:
@PricelessRabbit please, update your branch

PricelessRabbit commented:
OMG sorry for the delay. I updated the branch. It's ok or i have to do some changes to the doc?

manuio commented:
> OMG sorry for the delay. I updated the branch. It's ok or i have to do some changes to the doc?

@PricelessRabbit looks like you have some unresolved remarks. @mteodor is this PR ok?

PricelessRabbit commented:
> > OMG sorry for the delay. I updated the branch. It's ok or i have to do some changes to the doc?
> 
> @PricelessRabbit looks like you have some unresolved remarks. @mteodor is this PR ok?

Yeah at the time i changed the url in the doc to match the one it was working for me. I don't know if it is still valid or it is changes as the remarks  (the `http://mainflux.com:8204` was not working)

drasko reviewed (approved):
LGTM

drasko commented:
Merged!

Thanks @PricelessRabbit
